Shared/Connect-M365.ps1

```diff
@@ -0,0 +1,159 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+function Test-EXOConnection {
+    [CmdletBinding(SupportsShouldProcess, ConfirmImpact = 'High')]
+    [OutputType([bool])]
+    param (
+        [Switch]$Force
+    )
+    #Validate EXO V2 is installed
+    if ((Get-Module -ListAvailable | Where-Object { $_.Name -like "ExchangeOnlineManagement" }).count -ge 1) {
+        Write-Host "ExchangeOnlineManagement Powershell Module installed"
```

Comment:
For the Test cmdlets, let's avoid writing to the screen of information. This should be done in Verbose information.

Reply:
This is still here, needs to be addressed.